### ISSUE TITLE
MultiActionButton: We cannot use NVDA for accessibility on this component 

### ISSUE THREAD

**Issue by jerlau** [Sage/carbon#5473]
### Current behaviour

NVDA is a software use to describe what you do on your computer (accessibility software for blind or visually impaired).
- Start NVDA.
- open https://carbon.sage.com/?path=/docs/multi-action-button--default-story
- Set the focus on one of MultiActionButton
- Try to use the keyboard to open the list of buttons.

the list of button is never opened.

### Expected behaviour

We can use NVDA for accessibility and open the list of buttons in the MultiActionButton component.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/docs/multi-action-button--default-story

### JIRA Ticket (Sage Only)

SBS-39772

### Carbon Version

110.7.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Anything else we should know?

NVDA is available only on windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
Hi @jerlau, what keyboard commands are you using to try and open the menu? Arrow keys appear to be functioning ok.

**Comment by Parsium** [comment]
Hi @jerlau, please could share which commands you used for this as @nicktitchmarsh described?

**Comment by jerlau** [comment]
@Parsium  Hi, 

After installing NVDA on Windows, It seems to work now. You can close this issues I think.
